### ISSUE TITLE
[bitnami/dokuwiki] - Permission denied when setting persistence when deploying helm chart

### ISSUE THREAD

**Issue by Vorkane** [bitnami/charts#3338]
**Which chart**:
Bitnami Dokuwiki - lastest

**Describe the bug**
When deploying the chart from helm it creates the container, pods, services, and pvc. But when it goes to persist the installation I get in the logs:

  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100  6403  100  6403    0     0   390k      0 --:--:-- --:--:-- --:--:--  390k
dokuwiki 14:50:22.75 INFO  ==> apache stopped
dokuwiki 14:50:22.76 INFO  ==> Persisting DokuWiki installation
cp: cannot create directory '/bitnami/dokuwiki/data': Permission denied

**To Reproduce**
1. helm install dokuwiki --set global.storageClass=ibmc-file-bronze bitnami/dokuwiki

**Expected behavior**

**Version of Helm and Kubernetes**:

- Output of `helm version`:

```
version.BuildInfo{Version:"v3.3.0-rc.1", GitCommit:"5c2dfaad847df2ac8f289d278186d048f446c70c", GitTreeState:"dirty", GoVersion:"go1.14.4"}
```

- Output of `kubectl version`: 

```
Client Version: version.Info{Major:"1", Minor:"18", GitVersion:"v1.18.6", GitCommit:"dff82dc0de47299ab66c83c626e08b245ab19037", GitTreeState:"clean", BuildDate:"2020-07-15T16:58:53Z", GoVersion:"go1.13.9", Compiler:"gc", Platform:"linux/amd64"}
Server Version: version.Info{Major:"1", Minor:"18", GitVersion:"v1.18.6+IKS", GitCommit:"2da45c03415d4c5b0661d244fd1be563554b0f26", GitTreeState:"clean", BuildDate:"2020-07-15T21:03:28Z", GoVersion:"go1.13.9", Compiler:"gc", Platform:"linux/amd64"}
```

**Additional context**

**Comment by dani8art** [comment]
Hi @Vorkane, thanks for opening the issue.

It seems we are missing some init-container to ensure persistence volumes have the right permissions, I going to try something, will get back to you soon.

**Comment by eancuta** [comment]
@Vorkane does the persistent directory you are trying to use have write permissions for the user? I know for docker that the persistent directory needs to give access to user 1001 and that the user has write permissions.

**Comment by dani8art** [comment]
Hi @Vorkane we have added an init-container which will ensure that the volume has the proper permissions, please, let's move the conversation there! #3441 thanks!

**Comment by Vorkane** [comment]
> @Vorkane does the persistent directory you are trying to use have write permissions for the user? I know for docker that the persistent directory needs to give access to user 1001 and that the user has write permissions.

Through IKS I was using the file storageClass, I don't have control over the backend storage. I had tried with both the 1001, and forcing 0. Both  did not work.

Though I did get something working, I setup block storageClass in IKS and then deployed on the block storageClass and that ended up deploying with no problems and I have been able to work DokuWiki.
